### PR TITLE
SUBMARINE-269. Update Docs After Spin-off

### PR DESCRIPTION
**Pull request by wangdatan**
### What is this PR for?

Update docs after spin-off

### What type of PR is it?
Documentation

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-269

### How should this be tested?

First time? Setup Travis CI as described on https://submarine.apache.org/contribution/contributions.html#continuous-integration
Strongly recommended: add automated unit tests for any new or changed behavior
Outline any manual steps to test the PR here.

### Questions:

Does the licenses files need update? No
Is there breaking changes for older versions? No
Does this needs documentation? N/A

### CODE REVIEW

**Comment by wangdatan** [comment]
@liuxunorg , @yuanzac can you help to check the new one?

**Comment by jojochuang** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!---
+# <!---
```
Comment (liuxunorg) [inline README.md]: wrong format, Not need `#`

Comment (wangdatan) [reply README.md]: @liuxunorg thanks for catching this issue, just updated.

**Comment by liuxunorg** [comment]
Will merge if no more comment.

**Pull request closed without merging**